Utilities/DataFlow/CMakeLists.txt

```diff
@@ -29,24 +34,31 @@ set(LIBRARY_NAME ${MODULE_NAME})
 set(BUCKET_NAME ${MODULE_BUCKET_NAME})
 
 O2_GENERATE_LIBRARY()
+target_compile_options(${LIBRARY_NAME} PUBLIC -O0 -g)
```

Comment:
Agree. We really need to have way to build only tests and related libraries in debug mode, though. @dberzano any ideas regarding this?